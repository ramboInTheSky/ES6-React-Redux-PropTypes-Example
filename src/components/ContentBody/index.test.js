import React from 'react';
import { shallow } from 'enzyme';

import ContentBlock from './';

describe('<ContentBlock />', () => {
  it('renders correctly', () => {
    expect(shallow(<ContentBlock>Hello</ContentBlock>)).toMatchSnapshot();
  });
});
