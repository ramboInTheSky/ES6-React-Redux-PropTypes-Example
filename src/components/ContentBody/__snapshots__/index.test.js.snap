// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ContentBlock /> renders correctly 1`] = `
<styled.div
  className="container-fluid"
>
  <styled.div
    className="container"
  >
    Hello
  </styled.div>
</styled.div>
`;
